src/dbnode/server/server.go

```diff
@@ -409,6 +413,11 @@ func Run(runOpts RunOptions) {
 		}
 	}
 
+	// TODO: actually use the mmap reporter
+	mmapReporter := newMmapReporter(scope)
+	go mmapReporter.Run(context.Background())
```

Comment:
This context needs to be closed or else the mmap reporter never closes.

Just need to do this I think:
```go
	mmapReporter := newMmapReporter(scope)
	mmapReporterCtx, cancel := context.WithCancel(context.Background())
	defer cancel()
	go mmapReporter.Run(mmapReporterCtx)
```

Reply:
Ah, thought it was okay cuz the it would just run until the server terminates.